xbmc/cores/RetroPlayer/cheevos/Cheevos.h

```diff
@@ -32,6 +34,12 @@ class CCheevos
   void EnableRichPresence();
   std::string GetRichPresenceEvaluation();
 
+  void ActivateAchievement();
+  static void Callback_URL_ID(const char* achievementUrl, unsigned int cheevoId);
+  void CheckTriggeredAchievement();
+
+  static std::unordered_map<unsigned, std::vector<std::string>> m_activatedCheevoMap;
```

Comment:
Actually, because @Shardul555 did such a good job addressing our other comments in both https://github.com/garbear/xbmc/pull/126 and https://github.com/garbear/xbmc/pull/127 that my only comment really left is about this unpleasant static.

It's easy to address, but we might want to modify the rcheevos to handle instances like @AlwinEsch did for our add-ons.